src/main.tsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import App from './App.tsx';
-import { BrowserRouter } from 'react-router-dom';
+import theme from './theme/index.ts';
```

Comment:
```suggestion
import theme from './theme/';
```

이렇게 해도 작동이 될 것 같은데 한 번 확인해주실 수 있으실까요??